### ISSUE TITLE
Fluorescence exposure time

### ISSUE THREAD

**Issue by kyoohkim** [BrillouinMicroscopy/BrillouinAcquisition#125]
Exposure time for fluorescence mode in the Erlangen setup is limited up to 7 ms and cannot be set above 7 ms.

**Comment by raimund-schluessler** [comment]
@kyoohkim Have you checked whether you can freely adjust the exposure time with the vendor software?

**Comment by kyoohkim** [comment]
Yes, I can adjust the exposure time with the vendor software freely. 
If I turn off the vendor software with this exposure time setting, this setting is saved and I can initialize again with the same setting in the vendor software. However, if I initialize Brillouin Acquisition software, the exposure time is set as 7 ms and cannot be set above.
